Scientific documentation for SRW v3.0.0 release (#257)
ufs-community/ccpp-physics: physics/docs/pdftxt/srw3_phy_update.txt

/**
\page ccpp_phy Physics Update Summary

This page provides an overview of the major updates in CCPP Physics since UFS-SRW v2.2.0.

\b General \b Changes:
- New supported suites:  RRFS_sas and HRRR_gf suites 
- \ref CLM_LAKE_page in HRRR, HRRR_gf and RRFS_sas suites
- New \ref SMK_DUST in HRRR, HRRR_gf and RRFS_sas suites
- New photochem interface, which replace ozphys_2015 and h2ophys

> Suite RRFS_v1beta, which were supported with UFS-SRW v2.2.0, have been replaced with RRFS_sas.

Reviewer: ```suggestion
Suite RRFS_v1beta, which was supported with UFS-SRW v2.2.0, has been replaced with RRFS_sas.
```

\b GFS  \b Scale-aware \b SAS  \b Cumulus \b Schemes:

> The updates between GFSv16 and GFSv17 are carefully outlined in Bengtsson and Han (2024) \cite Bengtsson_2024. The main updates include:

Reviewer: Even though this wording is correct, I suggest this change to avoid user confusion with the GFS_v16 suite.

```suggestion
The updates since the previous release are carefully outlined in Bengtsson and Han (2024) \cite Bengtsson_2024. The main updates include:
```

> - Implementation of a positive definition mass-flux scheme and a method for removing the negative tracers (Han et al. 2022 \cite Han_et_al_2022)

Reviewer: ```suggestion
- Implementation of a positive definite mass-flux scheme and a method for removing the negative tracers (Han et al. 2022 \cite Han_et_al_2022)
```

- Introduction of environmental wind shear effect and subgrid TKE dependence in convection, to seek improvements in hurricane forecast prediction (Han et al. 2024 \cite Han_2024)
- Introduction of stricter convective initiation criteria to allow for more CAPE to build up to address a low CAPE bias in GFSv16 (Han et al. 2021 \cite han_2021)
- Reduction of convective rain evaporation rate to address a systematic cold bias near the surface in GFSv16 (Han et al. 2021 \cite han_2021)
- A scale adaptive solution of improving convection/radiation interaction is proposed and tested for SFS application to address too much cloud shielding and a drift in SST 

\b Grell-Freitas \b Scale \b and \b Aerosol \b Aware \b Convection \b Scheme:
- Update for aerosol-awareness (experimental and not supported)
- Scale-awareness turned off when explicit microphysics is not active anywhere in the column
- Convection is completely suppressed at grid points where the MYNN PBL sheme produces shallow convection
- Radar reflectivity considers mass flux PDF as well as whether scale-awareness is turned on at the gird point in equation

> - Add a new parameter \p gf_coldstart to control where GF will be cold started or not.

Reviewer: ```suggestion
- Add a new parameter \p gf_coldstart to control whether GF will be cold started or not.
```

- Small increase of buoyancy length scale in convective environment
- Patch for ensuring non-zero cloud fractions for all grid cells where cloud mixing ratio is greater than 1e-6 or ice mixing ratio is greater than 1e-9
- Exclude qs to calculate theta tendency

\b Subgrid-scale (\b SGS) \b Clouds \b Scheme:
- Bug fix for cloud condensate input into RRTMG radiation
- New code section for use with SAS cumulus scheme
- Cloud fraction now computed as a mix between the area-dependent form and the modified Chaboureau and Bechtold (2005) \cite Chaboureau_2005 form
- Adjusted limit for the boundary flux functions

\b Thompson \b Cloud \b Microphysics \b Scheme:
- Ice generation supersaturation requirement reduced from 0.25 to 0.15 to generate more ice at the upper 
levels and reduce the outgoing longwave radiation bias
- Cloud number concentration divided into two parts (over land and others). Number concentration over 
ocean reduced to a smaller number (50/\f$cm^3\f$) from its previous default (100/\f$cm^3\f$). Both changes were made to increase
the downward shortwave radiative flux and reduce the negative bias off coastal regions including the Southeast Pacific

> - To reduce the high bias in downward shortwave (DSW) radiative flux over land has been increased from 100/\f$cm^3\f$ to 150/\f$cm^3\f$. The convective 

Reviewer: ```suggestion
- To reduce the high bias in downward shortwave (DSW), radiative flux over land has been increased from 100/\f$cm^3\f$ to 150/\f$cm^3\f$. The convective 
```

- Small fixes to the minimum size of snow and collision constants

\b NSSL \b Cloud \b Microphysics \b Scheme:
- Updated with 3-moment option

\b Radiation
- Coupling of GOCART aerosols with radiation (iaer=2011)
- Convective cloud water (liquid water + ice water) added to the calculations of cloud water path and ice water path for radiation cloud properties

\b NoahMP \b Land \b Surface \b Model:
- Option for using the unified frozen precipitation fraction in NoahMP
- Diagnostic 2-meter temperature and humidity now based on vegetation and bare-ground tiles (new namelist option \a iopt_diag) 
- Bug fixes for GFS-based thermal roughness length scheme
- New soil color dataset introduced to improve soil albedo to reduce the large warm bias found in the Sahel desert
- Wet leaf contribution factor is included
- Leaf-area index now depends on momentum roughness length
- Soil thermal conductivity enhancement: To reduce persistent nighttime and early morning warm surface temperature bias in GFS models,

> a new lookup table developed from the PNNL soil carbon dataset and VIIRS surface type data is included, and the effects of soil organic matter (SOM) on soil thermal properties have been accounted for GFSv17 prototype tests show that including SOM results in better simulation of 

Reviewer: ```suggestion
a new lookup table developed from the PNNL soil carbon dataset and VIIRS surface type data is included, and the effects of soil organic matter (SOM) on soil thermal properties have been accounted for. GFSv17 prototype tests show that including SOM results in better simulation of 
```
